### DISCUSSION TITLE
How to add Fisheye camera to the scenes?

### DISCUSSION THREAD

**Discussion by shivaang12** [allenai/ai2thor#1122] [Q&A]
By default the robot model in the scenes has perspective camera, there is anyway I can modify this to replace or coincide the perspective camera with a fisheye camera?

Thanks in advance

**Comment by shiwalimohan** [comment]
Some advice on how to achieve this will be very useful.

**Comment by shivaang12** [comment]
I used this package to convert the perspective images to fisheye image

https://github.com/dadung/G2D
